### ISSUE TITLE
@keplr-wallet/types version is very old

### ISSUE THREAD

**Issue by Scipion** [strangelove-ventures/wadesta#174]
## @keplr-wallet/types version is not compatible with modern versions of the library

If you use `@keplr-wallet/types` in a project, the latest versions of the library are not compatible with the `v0.12.23` that currently graz is using.

Latest version of `@keplr-wallet/types` is `0.12.156` from 13/10/2024.

The version in use for latest graz (version `0.1.26` from 20/11/2024) is still `0.12.23` from 7/9/2023 (more than one year older and 133 versions of difference). Bearing in bind `@keplr-wallet/types` updates big changes (sometimes not compatible with previous versions) even for bug versions, that difference of versions is a lot.

## Solution liked

Update dependencies to the latests ones.

## Alternatives to considered

Move `@keplr-wallet/*` to peer dependencies if possible

## Additional context

Here an example of TS error when providing `ChainInfo` to `GrazProvider` component:

```js
<GrazProvider
    grazOptions={{
        chains: [myChainInfo],
    }}
>
// ...
</GrazProvider>
```

```
Type 'import("/.../node_modules/@keplr-wallet/types/build/chain-info").ChainInfo' is not assignable to type 'import("/.../node_modules/graz/node_modules/@keplr-wallet/types/build/chain-info").ChainInfo'.
  Types of property 'nodeProvider' are incompatible.
    Type '{ readonly name: string; readonly email?: string | undefined; readonly discord?: string | undefined; readonly website?: string | undefined; } | undefined' is not assignable to type '{ readonly name: string; readonly email: string; readonly website?: string | undefined; } | undefined'.
      Type '{ readonly name: string; readonly email?: string | undefined; readonly discord?: string | undefined; readonly website?: string | undefined; }' is not assignable to type '{ readonly name: string; readonly email: string; readonly website?: string | undefined; }'.
        Types of property 'email' are incompatible.
          Type 'string | undefined' is not assignable to type 'string'.
            Type 'undefined' is not assignable to type 'string'.ts(2322)
```

**Comment by codingki** [comment]
yes we will update this

**Comment by codingki** [comment]
fixed in graz v0.1.28
